pkg/release/release.go

```diff
@@ -34,6 +34,32 @@ const (
 	bazelVersionPath  = "bazel-genfiles/version"
 	dockerVersionPath = "version"
 	tarballExtension  = ".tar.gz"
+
+	// GCSStagePath is the directory where release artifacts are staged before
+	// push to GCS.
+	GCSStagePath = "_output/gcs-stage"
+
+	// ReleaseStagePath is the directory where releases are staged.
+	ReleaseStagePath = "_output/release-stage"
+
+	// GCEPath is the directory where GCE scripts are created.
+	GCEPath = "_output/release-stage/full/kubernetes/cluster/gce"
+
+	// NewGCIPath is the GCI path used post 1.3 release.
+	NewGCIPath = "_output/release-stage/full/kubernetes/cluster/gce/gci"
+
+	// OldGCIPath is the GCI path used pre 1.3 release.
```

Comment:
Drop this. 1.3 has been out of support for a long time, so we shouldn't carry forward any semantics related to it.